packages/core/src/features/cancerDistribution/cnvPlot.ts

```diff
@@ -230,8 +231,11 @@ const slice = createSlice({
           state = castDraft(initialState);
           state.status = "rejected";
           state.error = response.errors.message;
+          return state;
         }
 
+        if (state.requestId != action.meta.requestId) return state;
```

Comment:
I think this check needs to be the first statement of the case.  As it currently is, an error from an earlier response would be made available to the application while we're waiting for the latest request to resolve.  See my earlier comment in the annotations slice about handling the `rejected` case.  I think the same reasoning applies to handling errors in fulfilled requested.

this comment applies to every reducer that is using requestId to process only the latest request and is handling errors within the fulfilled case.